cfgov/unprocessed/apps/teachers-digital-platform/js/search.js

```diff
@@ -82,8 +78,10 @@ function clearFilter(event) {
   if (target !== 'svg' && target !== 'path') {
     return;
   }
-  target = closest(event.target, '.a-tag');
-  const checkbox = queryOne(`${target.getAttribute('data-value')}`);
+  target = event.target.closest('.a-tag');
+  const checkbox = document.querySelector(
+    `${target.getAttribute('data-value')}`
```

Comment:
Why is this wrapped in a template literal string? Can't it be straight `target.getAttribute('data-value')`?